package/MDAnalysis/analysis/md.bib

```diff
@@ -0,0 +1,23 @@
+@article{[Best2013],
+		author= {RB Best, G Hummer, and WA Eaton,},
+		title = {"Native contacts determine protein folding mechanisms in atomistic simulations"}, 				
+		 _PNAS_ **110** 
```

Comment:
This is not part of the bibtex entry which needs to be formatted in a `field = {value}` style. It might be easier to click on the DOI URL and just download a bibtex entry from there.